scripts/xtensa-build-zephyr.py

```diff
@@ -185,6 +185,8 @@ def parse_args():
 						default=[], help="Paths to overlays")
 	parser.add_argument("-p", "--pristine", required=False, action="store_true",
 						help="Perform pristine build removing build directory.")
+	parser.add_argument("-s", "--perf", required=False, action="store_true",
+						help="enable performance build for core/task/module.")
```

Comment:
There are only 26 letters and I don't think we should spend one for each overlay. It barely saves any typing compared to `-o app/perf<TAB>` and it's an overlay that few people (if any) will use regularly.

Reply:
performance build is important, and for convinence, let's first use -s first, current only 7 chars are used, in future, if there is a  
necessary, let's combine or use -o, for now, let's keep it, since this is the first time for enabling.

Reply:
For the moment it's important... to you. Once enough people complain that typing `-o app/perf<TAB>` takes too much of their time then we can discuss again. This seems unlikely to me; I expect most people to use the "up" arrow most of the time.